docs/experiment/experiment_introduction.md

```diff
@@ -0,0 +1,23 @@
+# Experiment Introduction
+
+After a successful login you will be redirected to your personal landing page.
+From there, [navigate](../project/project_introduction.md#project-navigation) into your project of interest to see the project summary.
+In the project summary view you can either [navigate](#experiment-navigation) into an experiment
+or [create a new experiment](experiment_creation.md).
+## Experiment Navigation
+
+From the project summary you can navigate into your experiment via the experiment list.
+To do so, click on the name of the experiment you want to navigate into 
+in the experiment list on the top right.
+![project_summary](../project/images/project_summary.png)
+
+Alternatively you can also select your experiment of interest via clicking on its name in the 
+application drawer to the left
```

Comment:
```suggestion
application drawer to the left. All of the project's experiments are also available when you are already in one of the experiments.
```
I think this would be helpful information, but I am not sure if "in one of the experiments" is the best way to say this.